src/tools/metabase/index.js

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env node
+'use strict';
+
+const cli = require('commander');
+const { execute } = require('../utils');
+
+cli.description('Data pour metabase').parse(process.argv);
+
+execute(__filename, async ({ logger, db, Sentry }) => {
+  logger.info('Récupération des différentes données nécessaires au metabase public...');
+
+  const postesValidesDepartement = [];
+  const postesValidesStructure = [];
+  const conseillersRecrutesDepartement = [];
+  const conseillersRecrutesStructure = [];
+  const candidats = [];
+  const structuresCandidates = [];
+
+  /* Nombre de postes validés par département */
+  const queryPosteValidesDepartement = [
+    { $match: { 'statut': 'VALIDATION_COSELEC', 'coselec': { '$elemMatch': { 'avisCoselec': 'POSITIF' } } } },
+    { $unwind: '$coselec' },
```

Comment:
@MorganP-projects Pour ma compréhension, est-ce qu'il y aurait un résultat différent si on me met pas ce unwind ?

Reply:
On ne peut pas faire de $sum sur un champ du coselec, s'il n'y a pas de $unwind :)

Reply:
@MorganP-projects du coup pour obtenir le bon nombre de conseiller validé, je ne sais pas si va suffire si y en a plusieurs en POSITIF ?
C'est déterminé par la fonction getCoselec dans l'api
  let coselecsPositifs = null;
  if ('coselec' in structure && structure.coselec !== null) {
    coselecsPositifs = structure.coselec.filter(c => c.avisCoselec === 'POSITIF');
  }
  // On prend le dernier
  return (coselecsPositifs !== null && coselecsPositifs.length > 0) ? coselecsPositifs.slice(-1).pop() : null;

Reply:
Après avoir testé l'agrégation a le même résultat ! :)

Reply:
@MorganP-projects OK :) du coup je ne sais pas si on peut avoir des surprises dans le futur avec les coselec suivants. 
@jystervinou qu'en penses-tu ? Est-ce qu'il faut privilégier ta requête pour déterminer cette partie coselec ou on peut laisse comme ça ?

Reply:
oui il faut absolution utiliser ma fonction, qui centralise la logique de détermination du "dernier bon" Coselec

Reply:
il y a des cas tordus.

Reply:
déjà parce qu'elle est juste (eh eh), et si on doit changer la logique, on n'aura pas à le répercuter ailleurs